src/main/java/com/royalrangers/controller/UserController.java

```diff
@@ -55,23 +56,25 @@ public ResponseResult getUserToApprove(@PathVariable("id") Long platoonId){
     }
 
     @PostMapping("/approve")
-    public ResponseResult approveUser(@RequestBody List<Long> ids) {
+    public ResponseResult approveUser(@RequestBody Map<String, Object> params) {
+        List <Long> ids = (List<Long>) params.get("ids");
         userService.approveUsers(ids);
         return ResponseBuilder.success("Users approved successfully.");
     }
 
     @PostMapping("/reject")
-    public ResponseResult rejectUser(@RequestBody List<Long> ids) {
+    public ResponseResult rejectUser(@RequestBody Map <String, Object> params) {
+        List<Long> ids = (List<Long>) params.get("ids");
         userService.rejectUsers(ids);
         return ResponseBuilder.success("Users disabled.");
     }
 
     @PutMapping
     @PreAuthorize("isAuthenticated()")
-    public ResponseResult updateAuthorizedUser(@RequestBody UserBean update) {
+    public ResponseResult updateAuthorizedUser(@RequestBody Map <String, Object> params) {
```

Comment:
Revert your changes. Look at above comment. Also, rename UserBean alias to more understandable one. For example `userUpdateInfo`.